packages/flutter/lib/src/widgets/pop_scope.dart

```diff
@@ -8,12 +8,52 @@ import 'framework.dart';
 import 'navigator.dart';
 import 'routes.dart';
 
+/// Deprecated, use [PopInvokedWithResultCallback] instead
+@Deprecated(
+  'Use PopInvokedWithResultCallback instead. '
+  'This feature was deprecated after v3.22.0-12.0.pre.',
+)
+typedef PopInvokedCallback = void Function(bool didPop);
```

Comment:
Nit: I worry about removing all of the docs on these things and replacing them with just the sentence about deprecation. Can you just leave the docs duplicated?